src/System.Linq.Expressions/src/Facade/System.Linq.Expressions.csproj

```diff
@@ -1,26 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <!-- Setting default TargetGroup before importing dir.prop -->
     <TargetGroup Condition="'$(TargetGroup)' == ''">net46</TargetGroup>
   </PropertyGroup>
+
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Linq.Expressions</AssemblyName>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
-    <ProjectJson>..\project.json</ProjectJson>
```

Comment:
As part of our follow-up to merge the csproj's we should also try merging the project.json files as much as we can as well.